Defra.Cdp.Backend.Api/Endpoints/MigrationEndpoints.cs

```diff
@@ -0,0 +1,93 @@
+using Defra.Cdp.Backend.Api.Services.Github;
+using Defra.Cdp.Backend.Api.Services.Migrations;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Defra.Cdp.Backend.Api.Endpoints;
+
+public static class MigrationEndpoints
+{
+    public static void MapMigrationEndpoints(this IEndpointRouteBuilder app)
+    {
+        app.MapGet("/migrations/available", ListServicesWithMigrations);
```

Comment:
is /migrations/services more appropriate? Since its returning the services, and not the migrations? Endpoint is a bit misleading

Reply:
yeah, the route names aren't great.

Could go with the convention used in the test endpoints? 
````
/migrations  // list all services
/migrations/{name} // list available for service
/migration-runs 
/migration-runs/{id} 
```